testkit-backend/backend.go

```diff
@@ -608,7 +606,6 @@ func (b *backend) handleRequest(req map[string]interface{}) {
 				"Optimization:ConnectionReuse",
 				"Optimization:ImplicitDefaultArguments",
 				"Optimization:PullPipelining",
-				"Temporary:ConnectionAcquisitionTimeout",
```

Comment:
@robsdedude this PR unfortunately conflicts with timeout configuration on the testkit backend side
if this PR is merged, timeout/deadlines/... would be configured through each API via `context.Context` object, so it won't be possible to set a timeout before calling the related operation

Reply:
Couldn't you store the timeout that TestKit requires somewhere in you TestKit back end to then later pass it into a function call?

Reply:
actually, you're right, it should work